### DISCUSSION TITLE
Notes for 3 May 2021 meeting

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#34] [General]
### Done since last time (in reference to InKyung's notes on last meeting):
- missing definitions (issue #26): Flavio and InKyung proposed definitions, which were integrated in the HTML and Turtle files. Other parts of the issue are integrated elsewhere, so issue is closed:
	* recommendations on properties to use will be integrated in the specification 
	* new issue opened (#29) regarding links with the metadata glossary
- Governance document (issue #25): KSH proposed document outline, to discuss in dedicated agenda item
- Spanish translation (issue #24): provided by Juan. Issue closed. Must think to translate any new artifact (see note below)
- URI policy (issue #23): see note by Flavio and my answer. Policy and Turtle file changed, issue closed. Remaining question of English/American? See InKyung's message on domains reserved
- Scope (issue #22): Capability class introduced in the ontology. Check translations (Capacité, Capacidad) and need for definitions. Question on CSPA roles remaining. New issue (#32) opened on the need for building block.
- Closed Task issue (#21) and moved the question on CSA to a new issue (#33) assigned to Jenny

### Not done
- Issue #19: more time needed. Suggestion of StatisticalInformation and StatisticalEntity floating around.
- Issue #15: to discuss
- Illustrative example: to integrate in the specification 

### Note: missing definitions
- InternationalAgency, NationalStatisticalInstitute, StatisticalOrganization: French, Spanish
- Capability: English, French, Spanish

**Comment by ChLaaboudi** [comment]
Definition for **National Statistical Institute**
[EU Legislation EUR-LEX](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32009R0223&from=EN )

National statistical authority designated ~~by each Member State~~ as the body having the responsibility for coordinating all
activities at national level for the development, production and dissemination of ~~European~~ statistics (the NSI)

 FR institut national de statistiques 
 ES instituto nacional de estadística

**Comment by zoltanvereczkei** [comment]
I agree with the removal of "European" but I suggest to clarify that this is about official statistics. 
_... for the development, production and dissemination of **official** statistics"._

**Comment by FlavioRizzolo** [comment]
These are the definitions I added to the HTML spec a few weeks ago, adapted from the glossary of the Handbook on Management and Organization of National Statistical Systems provided by InKyung:

https://unstats.un.org/wiki/display/HSO/Glossary

**StatisticalOrganization**: Organization, or unit within an organization, whose primary role is the production of official statistics.

**NationalStatisticalInstitute**: The main producer of official statistics in a country and/or the organization responsible for coordinating all activities related to the development, production, and dissemination of official statistics in the national statistical system. The actual name given to the national statistical office in a country may be National Statistical Institute (NSI), National Bureau of Statistics (NBS), Central Bureau of Statistics (CBS), National Statistical Agency (NSA), Central Statistical Agency (CSA), Central Statistics Agency (CSA), etc.

**InternationalAgency**: A body with an international membership, scope, or presence whose primary role is the production of official statistics. There are many types of international organizations. One way of categorizing them is to distinguish between intergovernmental organizations, supranational organizations and international non-governmental organizations.

**Comment by FlavioRizzolo** [comment]
For ore info, please see https://github.com/linked-statistics/COOS/issues/26

**Comment by JALinnerud** [comment]
https://unstats.un.org/unsd/methodology/dataquality/un-nqaf-manual/

National statistical office (NSO): the leading statistical agency within a
national statistical system. National statistical office and national statistical
institute mean the same thing. In general, the NSO has a coordination
role within the national statistical system, and is responsible for the development,
production and dissemination of official statistics across multiple statistical
domains.

The definition is similiar to the one from OECD, but a bit more comprehensive.
